source/common/network/tcp_listener_impl.cc

```diff
@@ -111,15 +111,15 @@ void TcpListenerImpl::enable() {
   if (bind_to_port_) {
     socket_->ioHandle().enableFileEvents(Event::FileReadyType::Read);
   } else {
-    FANCY_LOG(debug, "The listener cannot be enabled since it's not bind to port.");
```

Comment:
hmm, should these be using this directly? I would expect us to have gone through `ENVOY_LOG` to reach the fancy log macro

Reply:
It is OK if `FINE_GRAIN_LOG/FANCY_LOG` macro is directly used. Same as bellow.

They are out of `--enable-fine-grain-logging` control and don't need to register as a logable component.

We can change them to ENVOY_LOG_MISC for now, but I am thinking future developers can still use `FINE_GRAIN_LOG` directly. Otherwise, we can say it is not recommended.

Reply:
Would we run into any issues doing `FANCY_LOG(...)` if we're not using the fancy logger e.g. would it not print? If that's the case then it would be a little confusing.

Reply:
It will still print the messages. The `--enable-fine-grain-logging` only controls the `ENVOY_LOG` macro to use fine grain logger or not.